src/AutoRest.CSharp/Mgmt/AutoRest/MgmtTarget.cs

```diff
@@ -270,18 +267,6 @@ private static void WriteExtensionFile(GeneratedCodeWorkspace project, MgmtClien
             AddGeneratedFile(project, $"Extensions/{extensionWriter.FileName}.cs", extensionWriter.ToString());
         }
 
-        private static bool ShouldSkipModelGeneration(TypeProvider model)
```

Comment:
we must have this method because previously the replaced models are represented by their respective type provider, for instance, if the model gets replaced and before the replacement, it is a `ResourceData`, we will construct a new ResourceData for it after the replacement.
But now, we always construct a `SystemObjectType` for them, therefore we no longer need a check whether to skip them or not.